lib/Conversion/CfToHandshake/CfToHandshake.cpp

```diff
@@ -107,7 +107,7 @@ mergeFuncResults(handshake::FuncOp funcOp, ConversionPatternRewriter &rewriter,
     return {};
 
   // Return values from multiple returns need to be merged together
-  SmallVector<Value, 4> results;
+  SmallVector<Value, 6> results;
```

Comment:
This is unrelated and unnecessary. The second optional template parameter to `SmallVector` is a vector pre-allocation size for on the stack, it cannot impact correctness. Please remove it.